Source/WebCore/dom/Document.cpp

```diff
@@ -4346,6 +4346,11 @@ bool Document::supportsPaintTiming() const
     return protectedSecurityOrigin()->isSameOriginDomain(topOrigin());
 }
 
+bool Document::supportsLargestContentfulPaint() const
+{
+    return settings().largestContentfulPaintEnabled();
+}
```

Comment:
I don't think this shortcut is really that much shorter, I would just inline `settings().largestContentfulPaintEnabled()` in the relevant places.